diff --git a/server.py b/server.py
index 2300393b..35d727e7 100644
--- a/server.py
+++ b/server.py
@@ -625,7 +625,13 @@ class PromptServer():
                     "python_version": sys.version,
                     "pytorch_version": comfy.model_management.torch_version,
                     "embedded_python": os.path.split(os.path.split(sys.executable)[0])[1] == "python_embeded",
-                    "argv": sys.argv
+                    "argv": sys.argv,
+                    "cwd": os.getcwd(),
+                    "base_path": folder_paths.base_path,
+                    "input_directory": folder_paths.get_input_directory(),
+                    "output_directory": folder_paths.get_output_directory(),
+                    "temp_directory": folder_paths.get_temp_directory(),
+                    "user_directory": folder_paths.get_user_directory(),
                 },
                 "devices": [
                     {
